examples/chatgpt/gpt_actions_library/gpt_action_snowflake_direct.ipynb

```diff
@@ -238,16 +238,48 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before you set up authentication in ChatGPT, please take the following steps in the application."
+    "Before you set up authentication in ChatGPT, please take the following steps in Snowflake.\n",
+    "\n",
+    "* Review the Snowflake documentation on [Network Policies](https://docs.snowflake.com/en/user-guide/network-policies)\n",
+    "* Go to the Snowflake Worksheets\n",
+    "* Create a network rule with the ChatGPT IP egress ranges listed [here](https://platform.openai.com/docs/actions/production/ip-egress-ranges)\n",
+    "* Create a corresponding Network Policy"
```

Comment:
I think pointing to the source of truth is important (providing the example below is quite handy, I would keep it, but saying to get the IPs from https://platform.openai.com/docs/actions/production)

Reply:
added 2 lines to the code block to indicate more clearly that it's an example